plugins/idea/pluginImpl/src/main/resources/META-INF/plugin.xml

```diff
@@ -64,7 +64,8 @@
         <!-- Add your project components here -->
     </project-components>
     <actions>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this line start at column 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)